index.html

```diff
@@ -3221,40 +3231,42 @@ <h3>Terms of Use</h3>
         <pre class="example nohighlight"
           title="Usage of the termsOfUse property by an issuer">
 {
-  "@context": [
-    "https://www.w3.org/ns/credentials/v2",
-    "https://www.w3.org/ns/credentials/examples/v2"
-  ],
-  "id": "http://university.example/credentials/3732",
-  "type": ["VerifiableCredential", "ExampleDegreeCredential"],
-  "issuer": "https://university.example/issuers/14",
-  "validFrom": "2010-01-01T19:23:24Z",
-  "credentialSubject": {
-    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
-    "degree": {
-      "type": "ExampleBachelorDegree",
-      "name": "Bachelor of Science and Arts"
+    "@context": [
+        "https://www.w3.org/ns/credentials/v2",
```

Comment:
JSON should use common 2 space indent style.

Reply:
Done